<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fight_fight</class>
 <widget class="QWidget" name="fight_fight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fight_fight</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/mirror/Resources/bg00.png);</string>
  </property>
  <widget class="QTextEdit" name="edit_display">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>391</width>
     <height>337</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>88</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出战斗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>88</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始战斗</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_auto">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>16</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>自动战斗</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_hp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>体力</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_mp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>48</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>魔法</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_hp">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_mp">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>48</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_concise">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>简洁模式</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_quick">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>64</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>快速战斗</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_role_head">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mirror.qrc">:/head/Resources/head/role_z_m_h.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_monster_head">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>16</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="mirror.qrc">:/head/Resources/head/role_z_m_h.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>184</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>体力</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>208</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>魔法</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_hp">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>20</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>99</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_mp">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>48</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>99</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_dc">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>304</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>304</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>336</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>魔法</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>368</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>道术</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>464</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>回血</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>432</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>魔防</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>物防</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>496</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>回蓝</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>528</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击间隔</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_mc">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>336</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_sc">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>368</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_ac">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_mac">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>432</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_rhp">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>464</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_rmp">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>496</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_role_interval">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>528</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_mac">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>432</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_rhp">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>464</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_55">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>304</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_59">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>432</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>魔防</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_rmp">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>496</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_58">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>400</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>物防</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_interval">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>528</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_dc">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>304</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_ac">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_mc">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>336</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_61">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>496</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>回蓝</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_56">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>336</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>魔法</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_62">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>528</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击间隔</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_60">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>464</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>回血</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>208</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>魔法</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>184</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>体力</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_role_hp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>184</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_role_mp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>208</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_monster_hp">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>184</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_monster_mp">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>208</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>464</y>
     <width>391</width>
     <height>89</height>
    </rect>
   </property>
   <property name="title">
    <string>状态</string>
   </property>
   <widget class="QCheckBox" name="checkBox_MultipleRep">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>64</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>双倍声望剩余</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_MultipleDrop">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>双倍暴率剩余</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_MultipleExp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>16</y>
      <width>95</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>双倍经验剩余</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_MultipleExp">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>16</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_MultipleDrop">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_MultipleRep">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>64</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar_role_exp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>565</y>
     <width>810</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>6</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>6</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_57">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>368</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>道术</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_monster_sc">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>368</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_boss">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>88</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>寻找BOSS</string>
   </property>
  </widget>
  <widget class="QLabel" name="edit_role_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>8</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>体力</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="edit_monster_name">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>8</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>体力</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="edit_role_level">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>体力</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="edit_monster_level">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>体力</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="edit_role_vocation">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>95</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 15pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>战</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="edit_monster_vocation">
   <property name="geometry">
    <rect>
     <x>783</x>
     <y>95</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 15pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>无</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mirror.qrc"/>
 </resources>
 <connections/>
</ui>
